README.md

```
@@ -248,6 +212,120 @@ We can use `c` to process nested types. Indent level is for passing to other con
+	PaginationParams struct {
+		Start *int `json:"start,omitempty" validate:"omitempty,gt=0"`
+		End   *int `json:"end,omitempty" validate:"omitempty,gt=0"`
+	} `validate:"pageParams"`
+	Search *string `json:"search,omitempty" validate:"identifier"`
+}
+
+customTagHandlers := map[string]zen.CustomFn{
+	"identifier": func(c *zen.Converter, t reflect.Type, validate string, indent int) string {
+		return ".refine((val) => !val || /^[a-z0-9_]*$/.test(val), 'Invalid search identifier')"
+	},
+	"pageParams": func(c *zen.Converter, t reflect.Type, validate string, indent int) string {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)